src/main/java/net/glowstone/block/itemtype/ItemFlintAndSteel.java

```diff
@@ -48,6 +48,10 @@ private boolean setBlockOnFire(GlowPlayer player, GlowBlock clicked, BlockFace f
             return true;
         }
 
+        if (clicked.getType() != Material.LEAVES && fireBlock.getRelative(BlockFace.DOWN).getType() == Material.AIR) {
```

Comment:
Shouldn't this check if the type is not flammable? There are more blocks than leaves that allow for this.